src/Serilog.Sinks.Loggly/Serilog.Sinks.Loggly.csproj

```diff
@@ -16,6 +16,8 @@
     <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <RootNamespace>Serilog</RootNamespace>
+    <FileVersion>5.2.0.0</FileVersion>
```

Comment:
What does this change in the package / assembly and should it be aligned with the Version Prefix element?